src/es/leercapitulo/build.gradle

```diff
@@ -0,0 +1,16 @@
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlinx-serialization'
+
+ext {
+    extName = 'LeerCapitulo'
+    pkgNameSuffix = 'es.leercapitulo'
+    extClass = '.LeerCapitulo'
+    extVersionCode = 1
```

Comment:
Should `extVersionCode` continue from when LeerCapitulo was removed from the old repository? Mostly to ensure a clean upgrade path for people still holding on to the extension for whatever reason.

Reply:
yuuuuup